internal/gengapic/testdata/empty_client_init.want

```diff
@@ -41,37 +58,40 @@ func NewClient(ctx context.Context, opts ...option.ClientOption) (*Client, error
 	if err != nil {
 		return nil, err
 	}
-	c := &Client{
+	client := Client{CallOptions: defaultCallOptions()}
+
+	c := &GrpcClient{
 		connPool:    connPool,
 		disableDeadlines: disableDeadlines,
-		CallOptions: defaultCallOptions(),
-
 		client: mypackagepb.NewFooClient(connPool),
+		CallOptions: &client.CallOptions,
 	}
 	c.setGoogleClientInfo()
 
-	return c, nil
+	client.internalClient = c
+
+	return client, nil
 }
 
 // Connection returns a connection to the API service.
 //
 // Deprecated.
-func (c *Client) Connection() *grpc.ClientConn {
+func (c *GrpcClient) Connection() *grpc.ClientConn {
 	return c.connPool.Conn()
 }
 
-// Close closes the connection to the API service. The user should invoke this when
-// the client is no longer required.
-func (c *Client) Close() error {
```

Comment:
I guess this is not breaking because `internalClient` is embedded, so `Client` gets the `Close` method from there?

Reply:
Exactly